pyproject.toml

```diff
@@ -108,6 +108,10 @@ source = ["evap", "tools"]
 
 [tool.coverage.report]
 omit = ["*/migrations/*", "*__init__.py"]
+exclude_also = [
+    # mypy asserts these cases are not possible
+    '\n.*assert_never(\(.*\))',
```

Comment:
The intention of the leading `\n` is so that the whole `else`-block is excluded, right? I would add that as a comment (wasn't intuitive to me).

What is the purpose of the capturing group? Isn't clear to me from the coverage docs.

Reply:
yes, it's https://coverage.readthedocs.io/en/latest/excluding.html#multi-line-exclusion-regexes

I tried to find a regex which excludes the whole else branch but my regex skills were to limited (maybe with some positive lookahead or so, but I did not really see the point there

Reply:
I think the newlines do exactly what we want, just the capturing groups are not necessary. This should work:
```regex
\n.*assert_never\(.*\)
```
(To understand the intention of the "weird" newline, just add a comment, something along the lines of
```python3
# newline disables coverage for the whole block, see
# https://coverage.readthedocs.io/en/latest/excluding.html#multi-line-exclusion-regexes
```